VideoApp/VideoApp/Video/Room/Room.swift

```diff
@@ -19,7 +19,7 @@ import TwilioVideo
 @objc class Room: NSObject {
     enum Update {
         case didStartConnecting
-        case didConnect
+        case didConnect(roomSID: String, accessToken: String)
```

Comment:
I advocated for the chat feature to use a completely separate endpoint to get token and conversation name so that chat would be completely decoupled from video on the client. But I compromised and have to expose these things and do a little coordination in the view model. 

At least the backend is creating the conversation and adding the user as a participant.